takwimu/views.py

```diff
@@ -51,6 +52,32 @@ def get_context_data(self, **kwargs):
         context.update(takwimu_topics(self.request))
         return context
 
+class ProfileView(TemplateView):
```

Comment:
Wagtail handles view directly from the models in `models/dashboard.py`. What's missing?

Reply:
I could selected country